packages/rich-text/src/Toolbar/components/EmbedEntityWidget.tsx

```diff
@@ -23,14 +23,17 @@ export const EmbedEntityWidget = ({ isDisabled, canInsertBlocks }: EmbedEntityWi
   const onCloseEntityDropdown = () => setEmbedDropdownOpen(false);
   const onToggleEntityDropdown = () => setEmbedDropdownOpen(!isEmbedDropdownOpen);
 
-  const [canAccessAssets, setCanAccessAssets] = useState(false);
+  // Hardcoded `true` value following https://contentful.atlassian.net/browse/DANTE-486
+  // TODO: refine permissions check in order to account for tags in rules
+  const [canAccessAssets] = useState(true);
```

Comment:
I think it's safe to get rid of the `canAccessAssets` and the `useEffect` below it.

Reply:
@IBratWurst advised me to keep the query for monitoring reasons. As soon as the access API would correctly identify tag-based permissions, we want to readd this logic.